src/promptflow-evals/promptflow/evals/synthetic/templates/prompt_qa_conversation.txt

```diff
@@ -0,0 +1,29 @@
+You're an AI that converts sentences from Text into QnA conversation between an interviewer and an interviewee

```

Comment:
will we use prompty for these prompt templates in future?

Reply:
yes, we can talk about that at the review for the future release